packages/dashboard/src/lib/votes-utils.ts

```
@@ -0,0 +1,321 @@
+    let key: string
+    
+    switch (period) {
+      case 'hour':
+        key = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}-${date.getHours()}`
+        break
+      case 'day':
+        key = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`
+        break
+      case 'week':
+        const weekStart = new Date(date)
+        weekStart.setDate(date.getDate() - date.getDay())
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)